.github/workflows/ci-tests.yml

```diff
@@ -73,6 +77,8 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
+          cache: yarn
+          cache-dependency-path: ./back/yarn.lock
```

Comment:
question: @douglasduteil I cannot find any reference of `cache-dependency-path` in https://github.com/actions/cache or https://docs.github.com/en/actions/reference/dependency-caching-reference.

We don't use anymore a key like

```
      - name: Cache Node.js dependencies
        uses: actions/cache@v3
        with:
          path: node_modules
          key: ${{ runner.os }}-node-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-node-
```

Reply:
@gdarquie 
It's because the cache is handle by https://github.com/actions/setup-node see https://github.com/actions/setup-node?tab=readme-ov-file#caching-global-packages-data